styles/widgets/common/list.less

```diff
@@ -33,6 +33,10 @@
     }
 }
 
+.dx-list-group-body {
+    overflow: hidden;
+}
+
```

Comment:
jQuery sets overflow="hidden" on animation start, and it deletes overflow and sets display="none" at the element after animation finish